src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div v-if="!isEditing">
+      <div>Email Address: {{contact.email}}</div>
+      <div>Phone: {{contact.phone}}   ext:{{contact.phoneExtension}}</div>
```

Comment:
Check with Scott but I think "ext" should not be displayed if phoneExtension isn't entered.